### PR TITLE
Adds template from cookiecutter pytest plugin

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
### Overview

Adds a skeleton for the project, generated by the [pytest template tool](https://github.com/pytest-dev/cookiecutter-pytest-plugin/blob/main/README.md). I believe it is better to have this skeleton merged to main first, so next PR modifications are clearly differentiated from the template.

Everything from the template should be fine, but you might want to review if there is anything that needs to be changed.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# https://www.appveyor.com/docs/build-environment/#python
+
+environment:
+  matrix:
+    - PYTHON: "C:\\Python35"
+      TOX_ENV: "py35"
+
+    - PYTHON: "C:\\Python36"
+      TOX_ENV: "py36"
+
```
Comment (MaxJPRey) [inline appveyor.yml:11]: @AlejandroFernandezLuces Most of the project (PyVista, Pyansys, Numpy...) are not supporting version prior to 3.7.
I think here we can get rid of those versions (3.5 and 3.6).
```suggestion
```

Comment (MaxJPRey) [inline appveyor.yml:11]: @AlejandroFernandezLuces Most of the project (PyVista, Pyansys, Numpy...) are not supporting version prior to 3.7.
I think here we can get rid of those versions (3.5 and 3.6).
```suggestion
```

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# For more information about tox, see https://tox.readthedocs.io/en/latest/
+[tox]
+envlist = py35,py36,py37,py38,pypy3,flake8
```
Comment (MaxJPRey) [inline tox.ini:3]:
```suggestion
envlist = py37,py38,pypy3,flake8
```

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding='utf-8').read()
+
+
+setup(
+    name='pytest-pyvista',
+    version='0.1.0',
+    author='PyAnsys',
```
Comment (MaxJPRey) [inline setup.py:17]: Should be PyVista here
```suggestion
    author='PyAnsys',
```

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    install_requires=['pytest>=3.5.0'],
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Framework :: Pytest',
+        'Intended Audience :: Developers',
+        'Topic :: Software Development :: Testing',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```
Comment (MaxJPRey) [inline setup.py:36]:
```suggestion
```

File: setup.py (hunk 3 above)
Comment (AlejandroFernandezLuces) [reply setup.py]: Fixed, I couldn't apply your suggestion because you wrote PyAnsys again 😅

Comment (MaxJPRey) [reply setup.py]: Fixed. Thanks. I did not apply my suggestion originally.

Comment (AlejandroFernandezLuces) [reply setup.py]: I fixed it in a previous commit, it is solved now 😄

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**